source/sass/patterns/molecules/contextual-data.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../mixins/decorations";
+@import "../../mixins/positioning";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../settings/baselinegrid";
+@import "../../settings/color";
+
+.contextual-data {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include border(block-end);
+  @include padding($baselinegrid-space-extra_small, block);
+  @include text-align(center);
```

Comment:
Will need to be changed when other things are put into `contextual-data`, but for now as it only contains one thing (the doi), simple text-alignment is enough here.